.github/workflows/gcp-test-deploy.yml

```diff
@@ -19,10 +19,14 @@ on:
       test_variables:
         required: true
         type: string
-      zebra_state_path:
+      root_state_path:
         required: false
         type: string
         default: '/zebrad-cache'
+      zebra_state_dir:
+        required: false
```

Comment:
Are these really optional? It seems it shouldn't work without them.

If they aren't, then all other invocations of `gcp-test-deploy.yml` will need to specify those

Reply:
Other test don't need a state, so we can safely set this as not required. If not set by a test that needs it, it will crash.